src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -630,3 +630,13 @@ legend.schemaform-label.schemaform-file-label {
   margin-top: 0;
   margin-bottom: 1em;
 }
+
+// highlighted outlines for accessibility
+[role=button]:focus,
```

Comment:
This is probably fine, but is this change in this PR intentionally? Seems like a bleed-over from another PR.

Reply:
We want to have the highlight on all of the form elements so I figure we can just roll them all into the shared schema form css for now